recipes/x86_64_level/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set number = 0 %}
+
+package:
+  name: microarch-level-split
+  version: {{ level }}
+
+build:
+  number: {{ number }}
+  string: {{ number }}_{{ microarchitecture }}
+
```

```suggestion
  noarch: generic

```
This is just a technicality for `conda-smithy` to avoid it checking if it needs to create build files for other platforms. (I.e., without it, rendering will take considerably longer.)